src/bin/genesis.rs

```diff
@@ -0,0 +1,68 @@
+use dotenv::dotenv;
+use ethers::types::U256;
+use kakarot_rpc::test_utils::katana::genesis::KatanaGenesisBuilder;
+use lazy_static::lazy_static;
+use reth_primitives::B256;
+use serde::Serialize;
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet_crypto::FieldElement;
+use std::{
+    collections::HashMap,
+    env::var,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+lazy_static! {
+    static ref GENESIS_FOLDER_PATH: PathBuf = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join(".katana");
+    static ref KAKAROT_CONTRACTS_PATH: PathBuf =
+        Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join("lib/kakarot/build");
+    static ref INCORRECT_FELT: String = "INCORRECT".to_string();
+    static ref COINBASE_ADDRESS: FieldElement = FieldElement::from(0x12345u32);
+    static ref SALT: FieldElement = FieldElement::from_bytes_be(&[0u8; 32]).expect("Failed to convert salt");
```

Comment:
Do we need to explain why we chose a salt equal to 0?

Reply:
is it important to explain? I'm not sure there is an explanation, we just don't use a salt.